server_installation/topics/operator/keycloak-backup-cr.adoc

```diff
@@ -1,5 +1,7 @@
 
 [[_backup-cr]]
+Backup CR is *deprecated* and could be removed in future releases.
```

Comment:
This note needs to be probably elsewhere as this file will be removed in #1253. But I'm not entirely sure where.